src/__tests__/reporter/startSuiteTestReporting.spec.ts

```diff
@@ -0,0 +1,106 @@
+/*
+ *  Copyright 2021 EPAM Systems
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+import RPReporter from '../../reporter';
+import { mockConfig } from '../mocks/configMock';
+import { mockedTestParams, RPClientMock } from '../mocks/RPClientMock';
+import { StartTestObjType } from '../../models';
+import { TEST_ITEM_TYPES } from '../../constants';
+
+describe('start reporting suite/test', () => {
+  const reporter = new RPReporter(mockConfig);
+  reporter.client = new RPClientMock(mockConfig);
+  reporter.launchId = 'tempLaunchId';
+
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  // @ts-ignore
+  reporter.onTestBegin(mockedTestParams);
```

Comment:
It will be better to provide the mock test data directly here, this will make the test easier to understand and reduce unnecessary file transitions.